flask_saml_sso/session.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+
+from flask_session import sessions
+from itsdangerous import BadSignature, want_bytes
+
+
+def get_session_interface(app):
+    config = app.config.copy()
+
+    config.setdefault('SESSION_SQLALCHEMY', None)
+    config.setdefault('SESSION_SQLALCHEMY_TABLE', 'sessions')
+    config.setdefault('SESSION_PERMANENT', True)
+    config.setdefault('SESSION_USE_SIGNER', False)
+    config.setdefault('SESSION_KEY_PREFIX', 'session:')
+
+    return ModifiedSqlAlchemySessionInterface(
+        app,
+        config['SESSION_SQLALCHEMY'],
+        config['SESSION_SQLALCHEMY_TABLE'],
+        config['SESSION_KEY_PREFIX'],
+        config['SESSION_USE_SIGNER'],
+        config['SESSION_PERMANENT']
+    )
+
+
+class ModifiedSqlAlchemySessionInterface(sessions.SqlAlchemySessionInterface):
```

Comment:
Maybe a more telling name, i.e. indicating the nature of the modification, would be better here.